src/main/java/com/conveyal/datatools/manager/jobs/OtpRunnerManifest.java

```diff
@@ -0,0 +1,28 @@
+package com.conveyal.datatools.manager.jobs;
+
+import java.util.List;
+
+public class OtpRunnerManifest {
```

Comment:
Needs javadoc. It should definitely mention the linkage to the otp-runner repo/project with URL.